tests/console/sshd.pm

```diff
@@ -94,45 +98,61 @@ sub run {
     assert_script_run "whoami | grep root";
 
     # Generate RSA key for SSH and copy it to our new user's profile
+    assert_script_run "echo 'sshd.pm: generating RSA key pair' | logger";
     assert_script_run "ssh-keygen -t rsa -P '' -C 'root\@localhost' -f ~/.ssh/id_rsa";
     assert_script_run "su -c \"ssh-keygen -t rsa -P '' -C '$ssh_testman\@localhost' -f /home/$ssh_testman/.ssh/id_rsa\" $ssh_testman";
     assert_script_run "install -m 0644 -o $ssh_testman ~/.ssh/id_rsa.pub /home/$ssh_testman/.ssh/authorized_keys";
     assert_script_run "cat /home/$ssh_testman/.ssh/id_rsa.pub >> /home/$ssh_testman/.ssh/authorized_keys";
 
-    # Test non-interactive SSH and after that remove RSA keys
+    # Test non-interactive SSH
+    assert_script_run "echo 'sshd.pm: testing non interactive mode' | logger";
     assert_script_run "ssh -4v $ssh_testman\@localhost bash -c 'whoami | grep $ssh_testman'";
 
     # Port forwarding (bsc#1131709 bsc#1133386)
+    assert_script_run "echo 'sshd.pm: testing port forwarding ~/.ssh' | logger";
     assert_script_run "( ssh -NL 4242:localhost:22 $ssh_testman\@localhost & )";
     assert_script_run "( ssh -NR 0.0.0.0:5252:localhost:22 $ssh_testman\@localhost & )";
     assert_script_run 'until ss -tulpn|egrep "4242|5252";do sleep 1;done';
     assert_script_run "ssh-keyscan -p 4242 localhost >> ~/.ssh/known_hosts";
     assert_script_run "ssh-keyscan -p 5252 localhost >> ~/.ssh/known_hosts";
 
-    assert_script_run "ssh -p 4242 $ssh_testman\@localhost whoami";
-    assert_script_run "ssh -p 5252 $ssh_testman\@localhost whoami";
-    assert_script_run "ssh -p 4242 $ssh_testman\@localhost 'ssh-keyscan -p 22 localhost 127.0.0.1 ::1 >> ~/.ssh/known_hosts'";
-    assert_script_run "ssh -p 4242 $ssh_testman\@localhost 'ssh-keyscan -p 4242 localhost 127.0.0.1 ::1 >> ~/.ssh/known_hosts'";
-    assert_script_run "ssh -p 4242 $ssh_testman\@localhost 'ssh-keyscan -p 5252 localhost 127.0.0.1 ::1 >> ~/.ssh/known_hosts'";
-
-    assert_script_run "ssh -p 4242 -tt $ssh_testman\@localhost ssh -tt $ssh_testman\@localhost whoami";
-    assert_script_run "ssh -t -o ProxyCommand='ssh $ssh_testman\@localhost nc localhost 4242' $ssh_testman\@localhost whoami";
+    assert_script_run "ssh -v -p 4242 $ssh_testman\@localhost whoami";
+    assert_script_run "ssh -v -p 5252 $ssh_testman\@localhost whoami";
+    assert_script_run "ssh -v -p 4242 $ssh_testman\@localhost 'ssh-keyscan -p 22 localhost 127.0.0.1 ::1 >> ~/.ssh/known_hosts'";
+    assert_script_run "ssh -v -p 4242 $ssh_testman\@localhost 'ssh-keyscan -p 4242 localhost 127.0.0.1 ::1 >> ~/.ssh/known_hosts'";
+    assert_script_run "ssh -v -p 4242 $ssh_testman\@localhost 'ssh-keyscan -p 5252 localhost 127.0.0.1 ::1 >> ~/.ssh/known_hosts'";
+
+    assert_script_run "echo 'sshd.pm: testing ssh proxy' | logger";
+    assert_script_run "ssh -v -p 4242 -tt $ssh_testman\@localhost ssh -tt $ssh_testman\@localhost whoami";
+    sleep 60;
```

Comment:
I am sure you just put that in for temporary debugging and have a coffee more while your test runs :D

Reply:
:D Yes, this will be removed.